### PR TITLE
docs(SRV): add jobs documentation MTA-3846

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by nerda-codes — APPROVED** [review]

File: faq/serverless-jobs.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+content:
+  h1: Serverless Jobs
+hero: assets/serverless-jobs.webp
+---
+
+<Concept>
+
+## What are Serverless Jobs?
+
+a
```
Comment (bene2k1) [inline faq/serverless-jobs.mdx:14]: Missing concepts

File: faq/serverless-jobs.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## What types of workloads are best suited for Serverless Jobs?
+
+Serverless Jobs is ideal for tasks such as data processing, analysis and various computational operations that can be executed in a non-interactive batch mode. You can execute short and long-running jobs.
+
+</Concept>
+
+<Concept>
+
+## How can I monitor the activity of my Serverless jobs?
```
Comment (thomas-tacquet) [inline faq/serverless-jobs.mdx:28]:
```suggestion
## How can I monitor the activity of my Serverless Jobs?
```

File: serverless/jobs/how-to/run-a-job.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The Serverless Jobs page displays.
+
+2. Click the name of the job you want to run. The job **Overview** tab displays.
+
+3. Click the **Run job** button. A new entry appears in the **Job runs** section.
+
+The **Job runs** section contains basic monitoring information for your jobs:
+
+- A status, that can be `queued`, `ongoing`, `canceled`, or `succeeded`
```
Comment (thomas-tacquet) [inline serverless/jobs/how-to/run-a-job.mdx:34]:
```suggestion
- A status, that can be `queued`, `running`, `canceled`, `succeeded` or `failed`
```

File: serverless/jobs/how-to/run-a-job.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- A unique identifier (UID)
```
Comment (thomas-tacquet) [inline serverless/jobs/how-to/run-a-job.mdx:35]:
```suggestion
- An unique identifier (UID)
```

**Review by bene2k1 — APPROVED** [review]

File: serverless/jobs/how-to/run-a-job.mdx (hunk 4 above)
Comment (Shillaker) [reply serverless/jobs/how-to/run-a-job.mdx:35]: Weirdly it is actually `A unique identifier`

You would say "a unique identifier", "a uniform", "a unicorn", but "an umbrella" or "an uncle". No idea what the rule is here, perhaps a non-native speaker knows better than me :sweat_smile:

File: serverless/jobs/quickstart.mdx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+tags: jobs serverless namespace
+dates:
+  validation: 2023-11-07
+  posted: 2023-11-07
+categories:
+  - serverless
+  - jobs
+---
+
+Scaleway's Serverless Jobs allow you to run recurring tasks in the cloud. A Job is composed of a job definition, and some job runs.
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:17]:
```suggestion
Scaleway's Serverless Jobs allow you to run recurring tasks in the cloud. A Job is composed of a job definition, which is used as a template to create job runs.
```

File: serverless/jobs/quickstart.mdx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+<Message type="note">
+  Make sure that you have [created a Container Registry namespace](/containers/container-registry/how-to/create-namespace/) and [pushed the latest Alpine Linux Docker image](/containers/container-registry/how-to/push-images/) to it.
+</Message>
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The Jobs page displays.
+
+2. Click **+ Create job**.
+
+3. Complete the following steps in the wizard:
+    - Select the **Scaleway** Container Registry.
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:43]:
```suggestion
    - Select the **External** Container Registry.
```

File: serverless/jobs/quickstart.mdx [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    - Select the appropriate **Registry namespace** from the drop-down list, and then select the `alpine` **container** and `latest` **tag**.
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:44]:
```suggestion
    - Enter `alpine:latest` as the image name.
```

File: serverless/jobs/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    - Enter a **name** or use the automatically generated one. The name can only contain lowercase alphanumeric characters and dashes.
+    - Enter a **description** (optional).
+    - Select the region in which your job will be created.
+    - Choose the smallest **resources** available to allocate to your job.
+    - In the **Advanced options**, set the following starup command:
+        ```
+        ls
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:51]:
```suggestion
        echo "Hello Serverless Jobs!"
```

File: serverless/jobs/quickstart.mdx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+        ```
+    <Message type="note">
+      Refer to [How to create a Job from the Scaleway Registry](ADD LINK!!!!!!) for more information on the different parameters.
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:54]: Don't forget to add this link

File: serverless/jobs/quickstart.mdx [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/) 
+  - You have [signed up to the private beta](https://www.scaleway.com/en/betas/#serverless-jobs) and received a confirmation email.
+</Message>
+
+## How to create a job definition
+
+You can create a job either from the Scaleway Container Registry or any other public container registry (Docker Hub, AWS Container registries, GitLab container registry, etc.).
+
+### From the Scaleway Container Registry
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:32]: I propose we keep this quick-start as simple as possible, just using an external registry for now, then link to the specific docs on Scaleway registry.

File: serverless/jobs/quickstart.mdx [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+## How to run a job
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The Jobs page displays.
+
+2. Click the name of the Job you want to run.
+
+3. From the **Overview** tab, click **Run job**.
+
+The execution appears in the **Job runs** section of the **Overview** tab once it is complete.
```
Comment (Shillaker) [inline serverless/jobs/quickstart.mdx:88]:
```suggestion
The execution appears in the **Job runs** section of the **Overview** tab along with its status.
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+categories:
+  - serverless
+  - jobs
+---
+
+<Lightbox src="serverless-decision-tree.webp" alt="" />
+
+## Differences between Functions, Containers and Jobs
+
+Serverless Jobs are designed for batch computing workloads particularly large-scale and high-throughput jobs, like data processing, image rendering, or complex computations.
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:21]:
```suggestion
Serverless Jobs are designed for batch computing workloads particularly large-scale asynchronous tasks, like data processing, image rendering, or complex computations.
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+Choose Scaleway Serverless Jobs:
+
+- for long running tasks and resource intensive applications
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:25]:
```suggestion
- for long running tasks and resource-intensive applications
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+- to handle a large number of jobs efficiently
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:26]:
```suggestion
- to handle a large number of jobs concurrently
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+- if you do not need to expose your service on a port
+- for asychronous processing
+
+<Message type="note">
+  Serverless Jobs need your applications to be containerized, and therefore provide more flexibility in terms of runtime environments.
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:31]:
```suggestion
  To run as a Serverless Job, your applications must be containerized, which offers you a lot of choice when configuring your runtime environments.
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+</Message>
+
+Choose Scaleway Serverless Functions:
+
+- for stateless and small workloads
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:36]:
```suggestion
- for stateless and short-lived workloads (<15 minutes)
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+- for event-driven tasks
+- for optimum performance in lightweight scenarios
+- to scale tasks automatically
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:39]:
```suggestion
- for auto-scaling applications
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+- for short execution time (< 15 min)
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:40]:
```suggestion
- for web-facing applications and web servers
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+
+<Message type="note">
+  Serverless Functions are limited to available runtimes.
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:43]:
```suggestion
  Serverless Functions can only be written in languages available as Serverless Function runtimes <INSERT LINK TO FUNCTIONS PAGE>
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 20]
```diff
@@ -0,0 +1,56 @@
...
+</Message>
+
+Choose Scaleway Serverless Containers:
+
+- for stateless or stateful containerized applications like  microservices, APIs, and more
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:48]:
```suggestion
- for stateless, short-lived applications (<15 minutes)
- for web-facing applications such as web servers, APIs and microservices
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+- for event-driven tasks
+- if you need more control than with Serverless Functions
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:50]:
```suggestion
- if your programming language is not supported as a Serverless Function runtime
- if you need more control over your runtime environment with Serverless Functions
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+- to scale tasks automatically
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:51]:
```suggestion
- for auto-scaling workloads
```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+- for short execution time (< 15 min)
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:52]:
```suggestion

```

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 24]
```diff
@@ -0,0 +1,56 @@
...
+
+<Message type="note">
+  Serverless Containers need your applications to be containerized, and therefore provide more flexibility in terms of runtime environments.
```
Comment (Shillaker) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:55]:
```suggestion
  To run as a Serverless Container, your application must be containerized, giving you a lot of flexibility to configure your runtime environments.
```

**Review by Shillaker — DISMISSED** [review]

File: serverless/jobs/concepts.mdx [hunk 25]
```diff
@@ -0,0 +1,76 @@
...
+
+Serverless Jobs is a fully managed solution that enables you to plan, schedule, queue and run batch processing workloads without worrying about provisioning and scaling. A job is a single instance of one of these batch processing workloads.
+
+</Concept>
+
+<Concept>
+
+## Job definition
+
+A Job Definition is a template for a Serverless Job. It contains all the information necessary to run the job, including the container image used, the resources allocated, and the command to execute. The definition acts as a blueprint for [job runs](#job-run).
```
Comment (bene2k1) [inline serverless/jobs/concepts.mdx:42]:
```suggestion
A Job definition is a template for a Serverless Job. It contains all the information necessary to run the job, including the container image used, the resources allocated, and the command to execute. The definition acts as a blueprint for [job runs](#job-run).
```

File: serverless/jobs/how-to/create-job-from-external-registry.mdx [hunk 26]
```diff
@@ -0,0 +1,50 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - Your container image is available without authentication
+</Message>
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The jobs page displays.
+
+2. Click **Create job**. The job creation wizard displays.
+
+3. Complete the following steps in the wizard:
+    - Select the **External** Container Registry.
```
Comment (bene2k1) [inline serverless/jobs/how-to/create-job-from-external-registry.mdx:31]:
```suggestion
    - Select the **External** container registry.
```

not sure about this one, but I guess it should be lowercase as it refers to an external registry

File: serverless/jobs/how-to/create-job-from-scaleway-registry.mdx [hunk 27]
```diff
@@ -0,0 +1,51 @@
...
+  - You have [created a Container Registry namespace](/containers/container-registry/how-to/create-namespace/) and [pushed a container image](/containers/container-registry/how-to/push-images/) to it
+</Message>
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The jobs page displays.
+
+2. Click **Create job**. The job creation wizard displays.
+
+3. Complete the following steps in the wizard:
+    - Select the **Scaleway** Container Registry.
+    - Select the required Container Registry Namespace from the drop-down list, and then select the container image and tag.
```
Comment (bene2k1) [inline serverless/jobs/how-to/create-job-from-scaleway-registry.mdx:32]:
```suggestion
    - Select the required Container Registry namespace from the drop-down list, and then select the container image and tag.
```
not sure again, but iirc Namespace does not have an uppercase N

File: serverless/jobs/quickstart.mdx [hunk 28]
```diff
@@ -0,0 +1,70 @@
...
+
+The execution appears in the **Job runs** section of the **Overview** tab.
+
+<Message type="note">
+  Refer to [How to monitor a job](/serverless/jobs/how-to/monitor-job/) to see the logs of the job you just executed.
+</Message>
+
+## How to delete a job
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The Jobs page displays.
```
Comment (bene2k1) [inline serverless/jobs/quickstart.mdx:68]:
```suggestion
1. Click **Jobs** in the **Serverless** section of the side menu. The jobs page displays.
```
not sure again

**Review by bene2k1 — APPROVED** [review]

File: serverless/jobs/reference-content/difference-with-functions-containers.mdx [hunk 29]
```diff
@@ -0,0 +1,55 @@
...
+- if you do not need to expose your service on a port
+- for asychronous processing
+
+<Message type="note">
+  To run as a Serverless Job, your applications must be containerized, which offers you a lot of choice when configuring your runtime environments.
+</Message>
+
+Choose Scaleway Serverless Functions:
+
+- for for stateless and short-lived workloads (< 15 minutes)
```
Comment (ldecarvalho-doc) [inline serverless/jobs/reference-content/difference-with-functions-containers.mdx:36]:
```suggestion
- for stateless and short-lived workloads (< 15 minutes)
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**